_pages/cv.md

```diff
@@ -4,7 +4,7 @@ permalink: /cv/
 title: cv
 nav: true
 nav_order: 5
-cv_pdf: example_pdf.pdf
+cv_pdf: http://alberteinstein.eurocv.eu/
```

Comment:
I think we should keep the local pdf file on the template.

Reply:
You're probably right. I just wanted to include it so that users know that there is an option to add links for resume as well.
Maybe we can add a comment to indicate that?

Reply:
I agree.

Reply:
Just missing this point here @pranjalagg. Also I just fixed the prettier issue on main. Rebase it and it should be fine now.

Reply:
Done.